dependency/retrieval/components/verifier.go

```diff
@@ -9,7 +9,7 @@ import (
 	"os"
 	"strings"
 
-	"golang.org/x/crypto/openpgp"
+	"github.com/keybase/go-crypto/openpgp"
```

Comment:
I see this repository hasn't been updated in a while either. Is it still preferable to use over https://github.com/golang/crypto?

Reply:
It is one of the given community forks in the thread deprecating the openpgp library

Reply:
for posterity: https://github.com/golang/go/issues/44226

Reply:
just FYI it looks like https://github.com/ProtonMail/go-crypto/commits/main/ is actively developed. I haven't checked whether it would work for this use-case though.